pyproject.toml

```diff
@@ -1,6 +1,119 @@
 [build-system]
-requires = ["setuptools>=46.4.0", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = ["flit_core >=3.5,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "jupyter_book"
+authors = [{name = "Executable Book Project", email = "executablebooks@gmail.com"}]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Topic :: Text Processing :: Markup",
+]
+keywords = [
+    "reproducible",
+    "science",
+    "environments",
+    "scholarship",
+    "notebook",
+]
+dynamic = ["description", "version"]
+requires-python = ">=3.7"
+dependencies = [
+    "click>=7.1,<9",
+    "docutils>=0.15,<0.18",
+    "jsonschema<4",
+    # Include Jupytext to ensure users have the right version, even if not strictly necessary
+    "jupytext>=1.11.2,<1.12", # markdown-it-py~=1 required and support from 1.11.2
```

Comment:
An aside - I noticed that we're pinning jupytext on a minor version rather than a major version. It's now on `1.13.7` - do we still need to keep this?

Reply:
Well, we kind of don't need the jupytext dependency at all.
The lower pin was added when markdown-it-py was only an extra of jupytext, so you could end up with non-matching versions, and jupytext would just fail to convert properly.
But now, markdown-it-py isa first-class dependency of jupytext: https://github.com/mwouts/jupytext/blob/8cfa66bef380bf6d54479a06b092da30dbc9a7c4/setup.py#L38